src/html.rs

```diff
@@ -1,15 +1,15 @@
 use crate::common::*;
 use maud::{html, DOCTYPE};
 
-pub(crate) fn wrap_body(body: Markup) -> Response<Body> {
+pub(crate) fn wrap_body(title: String, body: Markup) -> Response<Body> {
   let html = html! {
     (DOCTYPE)
     html lang="en" {
       head {
         meta charset="utf-8";
         meta name="viewport" content="width=device-width, initial-scale=1";
         title {
-          "agora"
+          (title)
```

Comment:
I was thinking that if Agora were second, it would make it easier to distinguish multiple open agora tabs:

```suggestion
          (title) " · Agora"
```